get-started.md

```diff
@@ -56,6 +64,9 @@ Tangram's scene import syntax ([documentation](https://mapzen.com/documentation/
 
 ```
 import: https://mapzen.com/carto/refill-style-no-labels/refill-style-no-labels.yaml
```

Comment:
We should only promote versioned assets. Suggest keeping with basic refill and adding:

`import: https://mapzen.com/carto/refill-style/6/refill-style.yaml`